src/test/kotlin/uk/gov/justice/digital/hmpps/whereabouts/integration/VideoLinkMigrateIntegrationTest.kt

```diff
@@ -97,7 +97,7 @@ class VideoLinkMigrateIntegrationTest : IntegrationTest() {
         VideoLinkBookingEventType.CREATE,
         VideoLinkBookingEventType.DELETE,
       )
-      assertThat(courtCode).isEqualTo("CVNTCC")
+      assertThat(courtCode).isEqualTo("UNKNOWN")
```

Comment:
Not sure I understand why this is `UNKNOWN`.  We will will have this from the original create event won't we?

Reply:
It's just the test data which sets up the event isn't putting in a court code - and when the event has nothing, it defaults it to UNKNOWN. I could put a court code here, if it makes you happy?

Reply:
Just wanted to make sure I wasn't missing something is all. Not always easy to tell with git change history on PR's.